README.md

```diff
@@ -1 +1,214 @@
+      <ul>
+        <li><a href="#setup">Setup</a></li>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#install">Install</a></li>
+        <li><a href="#usage">Usage</a></li>
+        <li><a href="#run-tests">Run tests</a></li>
+        <li><a href="#deployment">Deployment</a></li>
+      </ul>
+    </li>
+    <li><a href="#authors">👥 Authors</a></li>
+    <li><a href="#future-features">🔭 Future Features</a></li>
+    <li><a href="#contributing">🤝 Contributing</a></li>
+    <li><a href="#support">⭐️ Show your support</a></li>
+    <li><a href="#acknowledgements">🙏 Acknowledgements</a></li>
+    <li><a href="#faq">❓ FAQ</a></li>
+    <li><a href="#license">📝 License</a></li>
+  </ol>
+</details>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails React App <a name="about-project"></a>
+
+This **Rails React App** is integration of rails and react app.
+
+## 📹 Video <a name="video"></a>
+
+<div align="center">
+  <img width="700" src="recipe-app.gif" />
+</div>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Rails</a></li>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+    <li><a href="https://javascript.com/">Javascript</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+Implement a connection between a Ruby on Rails back-end and React front-end.
+Understand the pros and cons of different approaches of connecting Ruby on Rails back-end with React front-end.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Ruby
+- Rails
+- Node package manager (npm)
+
+### Setup
+
+#### Github
+- Enter this url: [https://github.com/stephnna/hello-rails-react](https://github.com/stephnna/hello-rails-react) in your web browser.
+- Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+- Select the download Zip option from drop-down menu.
+- Once the download is complete you will be able to access my project locally.
+
+#### Local (Terminal)
+
+```sh
+  git clone git@github.com:stephnna/hello-rails-react.git
+  cd hello-rails-react
+```
+
+### Install
+
+Install this project with:
+
+<!--
+
+```sh
+  cd budget-app
+  bundle install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `rails s` or `rails server` to start a development server.
+
```

Comment:
- [ ] **Required:** Kindly update this section with the proper usage commands.

Reply:
@brytebee,

Thank you for your review, I don't understand what you mean by '...proper usage commands' The app is actually run by using 'rails s' if you meant another thing please communicate if not kindly approve my project
Thanks!